src/js/jsx/sections/style/EffectsPanel.jsx

```diff
@@ -94,22 +94,24 @@ define(function (require, exports, module) {
             event.stopPropagation();
         },
 
-        shouldComponentUpdate: function (nextProps) {
-            if (this.props.disabled !== nextProps.disabled ||
-                this.props.active !== nextProps.active) {
-                return true;
-            }
-
-            if (!Immutable.is(this.props.document.layers.selected,
-                nextProps.document.layers.selected)) {
-                return true;
-            }
-
-            if (!nextProps.visible && !this.props.visible) {
+        shouldComponentUpdate: function (nextProps, nextState) {
+            // Only compare the existence of clipboard.
+            var clipboardChanged = !!this.state.clipboard !== !!nextState.clipboar,
```

Comment:
:boar: fixed. 

`clipboardChanged` and  `selectedLayerChanged` are used by the following `if` statement, so I think that's right timing to initial them.